trunk/source/WitnessParser/src/de/uni_freiburg/informatik/ultimate/witnessparser/yaml/Witness.java

```diff
@@ -53,7 +53,12 @@ public String toString() {
 	}
 
 	public boolean isCorrectnessWitness() {
-		// TODO: Check this, when we also support violation witnesses
+		// TODO: Consistency check with invariants
```

Comment:
This means "fail if both invariants and violation sequences are present", right?